### DISCUSSION TITLE
How do you actually use the file?

### DISCUSSION THREAD

**Discussion by sbesic-starke** [Tarikul-Islam-Anik/Animated-Fluent-Emojis#12] [Q&A] [answered]
When I download it as a PNG from your website or Microsoft, it's just a static PNG when I embed it... What am I missing?

Thank you in advance for any help

**Comment by Tarikul-Islam-Anik** [comment, marked as answer]
Hello, @abesic-starke. Hope you're doing well. 
Use image tag it'll work normally. For demo please check [here](https://github.com/Tarikul-Islam-Anik/tarikul-islam-anik/edit/main/README.md)

**Reply by sbesic-starke** [reply]
Thank you very much for your quick response Tarikul and thank you for making this Git public! I must've done something absolutely stupidly wrong, that it didn't work in my codebase - I don't know what. After trying the same thing again it just works now, sorry...😅
